common/src/main/java/tachyon/network/protocol/databuffer/DataBuffer.java

```diff
@@ -42,4 +42,11 @@
    * @return a read-only ByteBuffer representing the data
    */
   public abstract ByteBuffer getReadOnlyByteBuffer();
+
+  /**
+  * Release the underlying buffer of this DataBuffer if no longer needed.
+  *
+  * @return True if the underlying buffer is released or no need to release
+  */
+  public abstract boolean release();
```

Comment:
Is there any way to retain() the netty buffer once a DataNettyBuffer is created? If not, the reference count would always be 1, right before release() is called.